client/my-sites/sidebar/sidebar.jsx

```diff
@@ -361,17 +372,20 @@ module.exports = React.createClass( {
 
 		return (
 			<li className={ this.itemLinkClass( [ '/plans' ], linkClass ) }>
-				<a onClick={ this.trackUpgradeClick } href={ planLink }>
+				<a onClick={ this.trackUpgradeClick.bind( this, planLink ) } href={ planLink }>
 					<Gridicon icon="clipboard" size={ 24 } />
 					<span className="menu-link-text">{ this.translate( 'Plan', { context: 'noun' } ) }</span>
 				</a>
-				<a href={ planLink } className="plan-name" onClick={ this.trackUpgradeClick }>{ planName }</a>
+				<a href={ planLink } className="plan-name" onClick={ this.trackUpgradeClick.bind( this, planLink ) }>{ planName }</a>
 			</li>
 		);
 	},
 
-	trackUpgradeClick: function() {
-		analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', { cta_name: 'sidebar_upgrade_default' } );
+	trackUpgradeClick: function( planLink ) {
+		analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', {
+			cta_name: 'sidebar_upgrade_default',
+			path: planLink
```

Comment:
I don't think we can easily use this to segment the test, since it's dynamic based on site name (rather than just having two values). I'd probably avoid the bind entirely and just add another custom attribute with two different values based on the a/b-test value. Have you tried creating a funnel report in tracks yet? You might want to spend a little time playing with that to get a better feel for how to track these tests.